### DISCUSSION TITLE
Are UTM VMs automatically excluded from Time Machine?

### DISCUSSION THREAD

**Discussion by thomasokken** [utmapp/UTM#6851] [UTM Help (macOS)] [answered]
I would like to make sure my UTM VMs don't get backed up by Time Machine, but I can't figure out how to do that. They get created in ~/Library/Containers/com.utmapp.UTM/Data/Documents, but I can only see ~/Library/Containers/com.utmapp.UTM in Terminal, it's not visible in the Finder, even when I press Command-Shift-period. And when I go into Time Machine Settings -> Options, I can't add it to the exclude list, because it doesn't show up in the file selection box.

Are UTM VMs automatically excluded? I don't understand what is going on and why the VM directory appears to be hidden from me.

**Comment by osy** [comment, marked as answer]
In finder the folder shows up as UTM inside Library/Containers

**Reply by thomasokken** [reply]
Thanks!
